components/src/molecules/Tabs.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { css } from 'styled-components'
+import { TYPOGRAPHY, SPACING } from '../ui-style-constants'
+import { COLORS, BORDERS } from '../helix-design-system'
+import { POSITION_RELATIVE, DIRECTION_COLUMN, DIRECTION_ROW } from '../styles'
+import { Flex } from '../primitives'
+
+const defaultTabStyle = css`
```

Comment:
basically we use `SCREAMING_SNAKE_CASE` for a css var.

```suggestion
const DEFAULT_TAB_STYLE = css`
```

Reply:
updated

Reply:
lol i had no idea it was called `SCREAMING_SNAKE_CASE`